### PR TITLE
small addition for other currencies

### PR DESCRIPTION
**Pull request by LoisRP**
## High Level Overview of Change

Provide information about Accept headers for other currencies. 
https://app.asana.com/0/1159512795365018/1156328729357277/f

### Context of Change

We previously had information for several Accept headers, but no standard format for Accept headers for other currencies.

### Type of Change

- [x ] Documentation Updates

## Test Plan

Ran locally.

## Future Tasks
In this PR, refine the guidelines further.

### CODE REVIEW

File: docs/payid-headers.md [hunk 1]
```diff
@@ -27,7 +27,9 @@ Some different header options are shown here, with example values.
 | BTC      | `application/btc-mainnet+json`  | { address: '1BvBMSEYstWetAu4m4GFg7xJaNVN2' }                      |
 | XRP      | `application/xrpl-mainnet+json` | { address: "rw2ciyaNshpHe7bCHo4bRWq6pqqynnWKQg", tag: "67298042"} |
 | ACH      | `application/ach+json`          | { account: '363023456079',routing: '011302838'}                   |
-| All      | `application/payid+json`        | Variable depending on the contents of each address                |
+| All      | `application/payid+json`        | Variable, depending on the contents of each address               |
+
+If you create your own Accept header for another currency, follow the pattern for BTC Accept headers, and use three characters for the currency, as shown.
```
Comment (hbergren) [inline docs/payid-headers.md:32]: We should also ask them to make a PR to register it here?

@sappenin , is the docs website the right place to "register" a currency, or do we have/need a IANA-like list of networks and associated headers?

Comment (sappenin) [reply docs/payid-headers.md:32]: @hbergren Let's put this in the Docs site for now. Once it grows and becomes more widely-deployed, we can look to make an RFC, but for now this is a good place to located this.

Comment (sappenin) [inline docs/payid-headers.md:32]:
```suggestion
If you create your own Accept header for another currency, follow the pattern for BTC Accept headers, and use three characters for the currency, as shown. In addition, consider creating a PR to update this list.
```

Comment (aanchal4) [reply docs/payid-headers.md]: There already exists a IANA registration section in the PayID protocol RFC for Media-type. See section 14.2 of https://github.com/payid-org/rfcs/blob/master/dist/spec/payid-protocol.txt

Comment (sappenin) [reply docs/payid-headers.md]: > There already exists a IANA registration section in the PayID protocol RFC

Ah yes, I overlooked that. I think this should still live on the docs page, so maybe we adjust the note around "new types"?

@aanchal4 did you have a particular suggestion here?

Comment (ryangyoung) [reply docs/payid-headers.md]: FWIW, I opened https://github.com/payid-org/payid-documentation/pull/99 to capture @aanchal4's comment.

**Pull request merged**